subsys/jwt/jwt.c

```diff
@@ -160,7 +160,7 @@ static int jwt_add_header(struct jwt_builder *builder)
 #endif
 	int jwt_header_len = ARRAY_SIZE(jwt_header);
 
-	if (jwt_header_len > builder->len) {
+	if (jwt_header > builder->len) {
```

Comment:
This seems unrelated??

Reply:
good catch, no idea why clang-tidy is changing that, so, do not trust clang-tidy and double check.